packages/vite/src/module-runner/runner.ts

```diff
@@ -162,8 +162,16 @@ export class ModuleRunner {
     return false
   }
 
-  private isCurcularImport(importers: Set<string>, moduleId: string) {
+  private isCurcularImport(
```

Comment:
@hi-ogawa would you also update the name to be `isCircularImport` ?

Reply:
Oh, didn't notice. There's one more above `isCurcularModule`. I fixed that too.